### ISSUE TITLE
Questions about the design of the experimental task

### ISSUE THREAD

**Issue by ATP-BME** [zjc062/mind-vis#29]
After reading the paper, my understanding is that the pre-training task in this paper is designed to predict the signal at the same time point of the region that is masked out from the BOLD signal of the neighboring region. My question is why not design the task to recover the signal at the masked time point from the signal in the same region before and after the time point？Because of the high spatial correlation of the BOLD signal, recovering the signal at the same time point based on neighboring regions seems to be a relatively easy task.
